### ISSUE TITLE
2025-04-11, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#137]
# Today's Arxiv Papers

(1) [C3PO: Critical-Layer, Core-Expert, Collaborative Pathway Optimization for Test-Time Expert Re-Mixing](http://arxiv.org/abs/2504.07964v1), 2025-04-10

(2) [GLUS: Global-Local Reasoning Unified into A Single Large Language Model for Video Segmentation](http://arxiv.org/abs/2504.07962v1), 2025-04-10

(3) [MM-IFEngine: Towards Multimodal Instruction Following](http://arxiv.org/abs/2504.07957v1), 2025-04-10

(4) [VCR-Bench: A Comprehensive Evaluation Framework for Video Chain-of-Thought Reasoning](http://arxiv.org/abs/2504.07956v1), 2025-04-10

(5) [Scaling Laws for Native Multimodal Models Scaling Laws for Native Multimodal Models](http://arxiv.org/abs/2504.07951v1), 2025-04-10

(6) [Porting an LLM based Application from ChatGPT to an On-Premise Environment](http://arxiv.org/abs/2504.07907v1), 2025-04-10

(7) [Redefining Machine Translation on Social Network Services with Large Language Models](http://arxiv.org/abs/2504.07901v1), 2025-04-10

(8) [How do Large Language Models Understand Relevance? A Mechanistic Interpretability Perspective](http://arxiv.org/abs/2504.07898v1), 2025-04-10

(9) [Benchmarking Adversarial Robustness to Bias Elicitation in Large Language Models: Scalable Automated Assessment with LLM-as-a-Judge](http://arxiv.org/abs/2504.07887v1), 2025-04-10

(10) [An LLM-Driven Multi-Agent Debate System for Mendelian Diseases](http://arxiv.org/abs/2504.07881v1), 2025-04-10

(11) [Token Level Routing Inference System for Edge Devices](http://arxiv.org/abs/2504.07878v1), 2025-04-10

(12) [Pangu Ultra: Pushing the Limits of Dense Large Language Models on Ascend NPUs](http://arxiv.org/abs/2504.07866v1), 2025-04-10

(13) [Robust Hallucination Detection in LLMs via Adaptive Token Selection](http://arxiv.org/abs/2504.07863v1), 2025-04-10

(14) [2D-Curri-DPO: Two-Dimensional Curriculum Learning for Direct Preference Optimization](http://arxiv.org/abs/2504.07856v1), 2025-04-10

(15) [The KL3M Data Project: Copyright-Clean Training Resources for Large Language Models](http://arxiv.org/abs/2504.07854v1), 2025-04-10

(16) [Understanding Learner-LLM Chatbot Interactions and the Impact of Prompting Guidelines](http://arxiv.org/abs/2504.07840v1), 2025-04-10

(17) [Deceptive Automated Interpretability: Language Models Coordinating to Fool Oversight Systems](http://arxiv.org/abs/2504.07831v1), 2025-04-10

(18) [MOSAIC: Modeling Social AI for Content Dissemination and Regulation in Multi-Agent Simulations](http://arxiv.org/abs/2504.07830v1), 2025-04-10

(19) [Cluster-Driven Expert Pruning for Mixture-of-Experts Large Language Models](http://arxiv.org/abs/2504.07807v1), 2025-04-10

(20) [A System for Comprehensive Assessment of RAG Frameworks](http://arxiv.org/abs/2504.07803v1), 2025-04-10

(21) [FairEval: Evaluating Fairness in LLM-Based Recommendations with Personality Awareness](http://arxiv.org/abs/2504.07801v1), 2025-04-10

(22) [Plan-and-Refine: Diverse and Comprehensive Retrieval-Augmented Generation](http://arxiv.org/abs/2504.07794v1), 2025-04-10

(23) [Fairness Mediator: Neutralize Stereotype Associations to Mitigate Bias in Large Language Models](http://dx.doi.org/10.1145/3728881), 2025-04-10

(24) [Efficient Tuning of Large Language Models for Knowledge-Grounded Dialogue Generation](http://arxiv.org/abs/2504.07754v1), 2025-04-10

(25) [SF2T: Self-supervised Fragment Finetuning of Video-LLMs for Fine-Grained Understanding](http://arxiv.org/abs/2504.07745v1), 2025-04-10

(26) [Zero-Shot Cross-Domain Code Search without Fine-Tuning](http://dx.doi.org/10.1145/3729357), 2025-04-10

(27) [Automated Construction of a Knowledge Graph of Nuclear Fusion Energy for Effective Elicitation and Retrieval of Information](http://arxiv.org/abs/2504.07738v1), 2025-04-10

(28) [DeepGreen: Effective LLM-Driven Green-washing Monitoring System Designed for Empirical Testing -- Evidence from China](http://arxiv.org/abs/2504.07733v1), 2025-04-10

(29) [MRD-RAG: Enhancing Medical Diagnosis with Multi-Round Retrieval-Augmented Generation](http://arxiv.org/abs/2504.07724v1), 2025-04-10

(30) [PR-Attack: Coordinated Prompt-RAG Attacks on Retrieval-Augmented Generation in Large Language Models via Bilevel Optimization](http://arxiv.org/abs/2504.07717v1), 2025-04-10

(31) [Proactive User Information Acquisition via Chats on User-Favored Topics](http://arxiv.org/abs/2504.07698v1), 2025-04-10

(32) [FMNV: A Dataset of Media-Published News Videos for Fake News Detection](http://arxiv.org/abs/2504.07687v1), 2025-04-10

(33) [Synthetic Fluency: Hallucinations, Confabulations, and the Creation of Irish Words in LLM-Generated Translations](http://arxiv.org/abs/2504.07680v1), 2025-04-10

(34) [Unveiling the Impact of Multimodal Features on Chinese Spelling Correction: From Analysis to Design](http://arxiv.org/abs/2504.07661v1), 2025-04-10

(35) [Synthesizing High-Quality Programming Tasks with LLM-based Expert and Student Agents](http://arxiv.org/abs/2504.07655v1), 2025-04-10

(36) [On the Temporal Question-Answering Capabilities of Large Language Models Over Anonymized Data](http://arxiv.org/abs/2504.07646v1), 2025-04-10

(37) [Enhancing Large Language Models through Neuro-Symbolic Integration and Ontological Reasoning](http://arxiv.org/abs/2504.07640v1), 2025-04-10

(38) [Agent That Debugs: Dynamic State-Guided Vulnerability Repair](http://arxiv.org/abs/2504.07634v1), 2025-04-10

(39) [ConceptFormer: Towards Efficient Use of Knowledge-Graph Embeddings in Large Language Models](http://arxiv.org/abs/2504.07624v1), 2025-04-10

(40) [VLM-R1: A Stable and Generalizable R1-style Large Vision-Language Model](http://arxiv.org/abs/2504.07615v1), 2025-04-10

(41) [Boosting Universal LLM Reward Design through the Heuristic Reward Observation Space Evolution](http://arxiv.org/abs/2504.07596v1), 2025-04-10

(42) [REANIMATOR: Reanimate Retrieval Test Collections with Extracted and Synthetic Resources](http://arxiv.org/abs/2504.07584v1), 2025-04-10

(43) [Do LLMs Understand Your Translations? Evaluating Paragraph-level MT with Question Answering](http://arxiv.org/abs/2504.07583v1), 2025-04-10

(44) [Malware analysis assisted by AI with R2AI](http://arxiv.org/abs/2504.07574v1), 2025-04-10

(45) [Exploring Human-Like Thinking in Search Simulations with Large Language Models](http://arxiv.org/abs/2504.07570v1), 2025-04-10

(46) [Using LLMs for Analyzing AIS Data](http://arxiv.org/abs/2504.07557v1), 2025-04-10

(47) [AI-Slop to AI-Polish? Aligning Language Models through Edit-Based Writing Rewards and Test-time Computation](http://arxiv.org/abs/2504.07532v1), 2025-04-10

(48) [A taxonomy of epistemic injustice in the context of AI and the case for generative hermeneutical erasure](http://arxiv.org/abs/2504.07531v1), 2025-04-10

(49) [Automating the Path: An R&D Agenda for Human-Centered AI and Visualization](http://arxiv.org/abs/2504.07529v1), 2025-04-10

(50) [Supervised Optimism Correction: Be Confident When LLMs Are Sure](http://arxiv.org/abs/2504.07527v1), 2025-04-10

(51) [Why We Feel: Breaking Boundaries in Emotional Reasoning with Multimodal Large Language Models](http://arxiv.org/abs/2504.07521v1), 2025-04-10

(52) [VideoExpert: Augmented LLM for Temporal-Sensitive Video Understanding](http://arxiv.org/abs/2504.07519v1), 2025-04-10

(53) [Enhancements for Developing a Comprehensive AI Fairness Assessment Standard](http://dx.doi.org/10.1109/COMSNETS63942.2025.10885551), 2025-04-10

(54) [GPT Carry-On: Training Foundation Model for Customization Could Be Simple, Scalable and Affordable](http://arxiv.org/abs/2504.07513v1), 2025-04-10

(55) [Apt-Serve: Adaptive Request Scheduling on Hybrid Cache for Scalable LLM Inference Serving](http://dx.doi.org/10.1145/3725394), 2025-04-10

(56) [Kimi-VL Technical Report](http://arxiv.org/abs/2504.07491v1), 2025-04-10

(57) [UniCAIM: A Unified CAM/CIM Architecture with Static-Dynamic KV Cache Pruning for Efficient Long-Context LLM Inference](http://arxiv.org/abs/2504.07479v1), 2025-04-10

(58) [Defense against Prompt Injection Attacks via Mixture of Encodings](http://arxiv.org/abs/2504.07467v1), 2025-04-10

(59) [Enhanced Question-Answering for Skill-based learning using Knowledge-based AI and Generative AI](http://arxiv.org/abs/2504.07463v1), 2025-04-10

(60) [Achilles Heel of Distributed Multi-Agent Systems](http://arxiv.org/abs/2504.07461v1), 2025-04-10

(61) [Beyond LLMs: A Linguistic Approach to Causal Graph Generation from Narrative Texts](http://arxiv.org/abs/2504.07459v1), 2025-04-10

(62) [CyberAlly: Leveraging LLMs and Knowledge Graphs to Empower Cyber Defenders](http://arxiv.org/abs/2504.07457v1), 2025-04-10

(63) [How Can Objects Help Video-Language Understanding?](http://arxiv.org/abs/2504.07454v1), 2025-04-10

(64) [LoRI: Reducing Cross-Task Interference in Multi-Task Low-Rank Adaptation](http://arxiv.org/abs/2504.07448v1), 2025-04-10

(65) [Revisiting LLM Evaluation through Mechanism Interpretability: a New Metric and Model Utility Law](http://arxiv.org/abs/2504.07440v1), 2025-04-10

(66) [LLM4Ranking: An Easy-to-use Framework of Utilizing Large Language Models for Document Reranking](http://arxiv.org/abs/2504.07439v1), 2025-04-10

(67) [From Token to Line: Enhancing Code Generation with a Long-Term Perspective](http://arxiv.org/abs/2504.07433v1), 2025-04-10

(68) [LLM-Enabled Data Transmission in End-to-End Semantic Communication](http://arxiv.org/abs/2504.07431v1), 2025-04-10

(69) [Task-oriented Age of Information for Remote Inference with Hybrid Language Models](http://arxiv.org/abs/2504.07428v1), 2025-04-10

(70) [Enhancing Player Enjoyment with a Two-Tier DRL and LLM-Based Agent System for Fighting Games](http://arxiv.org/abs/2504.07425v1), 2025-04-10

(71) [Routing to the Right Expertise: A Trustworthy Judge for Instruction-based Image Editing](http://arxiv.org/abs/2504.07424v1), 2025-04-10

(72) [AgentAda: Skill-Adaptive Data Analytics for Tailored Insight Discovery](http://arxiv.org/abs/2504.07421v1), 2025-04-10

(73) [RadZero: Similarity-Based Cross-Attention for Explainable Vision-Language Alignment in Radiology with Zero-Shot Multi-Task Capability](http://arxiv.org/abs/2504.07416v1), 2025-04-10

(74) [Leveraging LLMs for Multimodal Retrieval-Augmented Radiology Report Generation via Key Phrase Extraction](http://arxiv.org/abs/2504.07415v1), 2025-04-10

(75) [AI Coding with Few-Shot Prompting for Thematic Analysis](http://arxiv.org/abs/2504.07408v1), 2025-04-10

(76) [Talking Point based Ideological Discourse Analysis in News Events](http://arxiv.org/abs/2504.07400v1), 2025-04-10

(77) [A Novel Mamba-based Sequential Recommendation Method](http://arxiv.org/abs/2504.07398v1), 2025-04-10

(78) [Automating quantum feature map design via large language models](http://arxiv.org/abs/2504.07396v1), 2025-04-10

(79) [Task-Circuit Quantization: Leveraging Knowledge Localization and Interpretability for Compression](http://arxiv.org/abs/2504.07389v1), 2025-04-10

(80) [TALE: A Tool-Augmented Framework for Reference-Free Evaluation of Large Language Models](http://arxiv.org/abs/2504.07385v1), 2025-04-10

(81) [Enhancing Time Series Forecasting via Multi-Level Text Alignment with LLMs](http://arxiv.org/abs/2504.07360v1), 2025-04-10

(82) [Revisiting Prompt Optimization with Large Reasoning Models-A Case Study on Event Extraction](http://arxiv.org/abs/2504.07357v1), 2025-04-10

(83) [Throughput-Optimal Scheduling Algorithms for LLM Inference and AI Agents](http://arxiv.org/abs/2504.07347v1), 2025-04-10

(84) [Code Generation with Small Language Models: A Deep Evaluation on Codeforces](http://arxiv.org/abs/2504.07343v1), 2025-04-09

(85) [Zeus: Zero-shot LLM Instruction for Union Segmentation in Multimodal Medical Imaging](http://arxiv.org/abs/2504.07336v1), 2025-04-09

(86) [Alice: Proactive Learning with Teacher's Demonstrations for Weak-to-Strong Generalization](http://arxiv.org/abs/2504.07316v1), 2025-04-09

(87) [PAYADOR: A Minimalist Approach to Grounding Language Models on Structured Data for Interactive Storytelling and Role-playing Games](http://arxiv.org/abs/2504.07304v1), 2025-04-09

(88) [Modeling Response Consistency in Multi-Agent LLM Systems: A Comparative Analysis of Shared and Separate Context Approaches](http://arxiv.org/abs/2504.07303v1), 2025-04-09

(89) [MDIT: A Model-free Data Interpolation Method for Diverse Instruction Tuning](http://arxiv.org/abs/2504.07288v1), 2025-04-09

(90) [RAISE: Reinforenced Adaptive Instruction Selection For Large Language Models](http://arxiv.org/abs/2504.07282v1), 2025-04-09

(91) [A Multi-Phase Analysis of Blood Culture Stewardship: Machine Learning Prediction, Expert Recommendation Assessment, and LLM Automation](http://arxiv.org/abs/2504.07278v1), 2025-04-09

(92) [Better Decisions through the Right Causal World Model](http://arxiv.org/abs/2504.07257v1), 2025-04-09

(93) [Acceptance Test Generation with Large Language Models: An Industrial Case Study](http://arxiv.org/abs/2504.07244v1), 2025-04-09

(94) [ConceptCarve: Dynamic Realization of Evidence](http://arxiv.org/abs/2504.07228v1), 2025-04-09

(95) [Leveraging Machine Learning Techniques in Intrusion Detection Systems for Internet of Things](http://arxiv.org/abs/2504.07220v1), 2025-04-09

(96) [SemEval-2025 Task 5: LLMs4Subjects -- LLM-based Automated Subject Tagging for a National Technical Library's Open-Access Catalog](http://arxiv.org/abs/2504.07199v1), 2025-04-09

(97) [Face-LLaVA: Facial Expression and Attribute Understanding through Instruction Tuning](http://arxiv.org/abs/2504.07198v1), 2025-04-09

(98) [HypoEval: Hypothesis-Guided Evaluation for Natural Language Generation](http://arxiv.org/abs/2504.07174v1), 2025-04-09

(99) [Sculpting Subspaces: Constrained Full Fine-Tuning in LLMs for Continual Learning](http://arxiv.org/abs/2504.07097v1), 2025-04-09

(100) [OmniCaptioner: One Captioner to Rule Them All](http://arxiv.org/abs/2504.07089v1), 2025-04-09
